Dynavity/Dynavity/view/canvas/CanvasThumbnailView.swift

```diff
@@ -2,20 +2,36 @@ import SwiftUI
 
 struct CanvasThumbnailView: View {
     var canvasName: String
+    var isSelected: Bool
     var body: some View {
         VStack {
-            // TODO: replace this with canvas thumbnail
-            Image(systemName: "doc")
-                .resizable()
-                .aspectRatio(contentMode: .fill)
-                .frame(width: 70)
+            if isSelected {
+                // TODO: replace this with canvas thumbnail
+                Image(systemName: "doc")
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .frame(width: 70)
+                    .overlay(checkSymbol)
```

Comment:
Does writing something like `.overlay(isSelected ? checkSymbol : nil)` instead work? To remove the code duplication

Reply:
Yep that works! Will change it, thanks:)